### ISSUE TITLE
Unable to rename the left pane tree elements of workbook window.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#387]
### Steps to Reproduce
1. Launch Xamarin Workbooks application. This will invokes "Create a new Workbook" window.
2. Create a Workbook with WPF platform. Navigate to Help-> Tutorial -> Hello Workbooks and invoke it.
3. Navigate to "Hello Workbooks" tree element on left side pane and rename "Hello workbooks" using keyboard.

### Expected Behavior
Should be able to rename left pane tree elements using Keyboard.
### Actual Behavior
Cannot able to rename "Hello Workbooks" tree element present in the left pane using keyboard.
Note: By using mouse, if user double clicks on "Hello workbooks" tree element, it invokes text edit box then user can rename it.

Inspect Behavior:
After double clicking on "Hello workbooks" tree element, it gives the edit text box. Please find the inspect screenshot with edit text box.
![screenshot 183](https://user-images.githubusercontent.com/22956130/40764376-672daccc-64c6-11e8-908b-2ce50de30120.png)

### User Impact:
Keyboard user cannot rename the left pane tree elements.
### Screenshot
Normal Screenshot:
![screenshot 184](https://user-images.githubusercontent.com/22956130/40764453-bb80f22a-64c6-11e8-97d8-d725f801bbac.png)

Screenshot of edit box for renaming tree element invoked by mouse.
![screenshot 182](https://user-images.githubusercontent.com/22956130/40764478-cd0ae51e-64c6-11e8-854b-e52a62efe81f.png)

### Environment
Windows : Version 1803 (OS Build : 17134.81)
Xamarin Version : 1.4(master/b3f92f9)

**Comment by WiproPSAccTest** [comment]
Please add the following labels to the bug:
A11yMAS
Accessibility
Xamarin-Workbooks-Windows
Wipro
MAS2.1.1

**Comment by AccessibilityTestingTeam-TCS** [comment]
Exemption Approved - 1813
